lib/Varien/File/Uploader.php

```diff
@@ -161,10 +161,10 @@ public function __construct($fileId)
         $this->_setUploadFileId($fileId);
         if (empty($this->_file['tmp_name']) || !file_exists($this->_file['tmp_name'])) {
             $errorCode = $this->_file['error'] ?? 0;
+            $code = empty($this->_file['tmp_name']) ? self::TMP_NAME_EMPTY : 0;
             if ($errorCode && isset(self::UPLOAD_ERRORS[$errorCode])) {
-                throw new Exception(self::UPLOAD_ERRORS[$errorCode]);
+                throw new Exception(self::UPLOAD_ERRORS[$errorCode], $code);
             }
-            $code = empty($this->_file['tmp_name']) ? self::TMP_NAME_EMPTY : 0;
             throw new Exception('File was not uploaded.', $code);
```

Comment:
To me, the message "File was not uploaded" is similar to `UPLOAD_ERR_NO_FILE => 'No file was uploaded',`, I wonder if it'd be better to just merge them together instead of throwing different Exceptions in two different cases 🤔, but not sure if any 3rd party code could be relying on that...